src/index.html

```diff
@@ -7,7 +7,56 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
+  <body class="body">
     <h1>Stars</h1>
+    <div class="stars">
+      <div class="stars--0">
+        <div class="stars_star_disabled"></div>
+        <div class="stars_star_disabled"></div>
+        <div class="stars_star_disabled"></div>
+        <div class="stars_star_disabled"></div>
+        <div class="stars_star_disabled"></div>
+      </div>
+      
+      <div class="stars--1">
```

Comment:
Add this class for every block
```suggestion
      <div class="stars stars--1">
```